.github/workflows/rust.yml

```diff
@@ -0,0 +1,32 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install Dependencies and Rustup
+      run: |
+        echo "Installing dependencies and Rustup..."
```

Comment:
Don't forget to remove this Rustup comment here too

Reply:
Will do.